src/core/initialize.cpp

```diff
@@ -740,6 +740,10 @@ void on_ghost_flags_change() {
   // VIRUTAL_SITES need v to update v of virtual sites
   ghosts_have_v = 1;
 #endif
+#ifdef DRUDE 
```

Comment:
After recent changes from Rudolf, this can  be set dynamically now. Please do so when the bond is actually activated. This will reduce the effect of merely compiling in the feature.